pkg/asset/manifests/operators.go

```diff
@@ -200,6 +201,7 @@ func (m *Manifests) generateBootKubeManifests(dependencies asset.Parents) []*ass
 		"03-openshift-web-console-namespace.yaml":          []byte(bootkube.OpenshiftWebConsoleNamespace),
 		"04-openshift-machine-config-operator.yaml":        []byte(bootkube.OpenshiftMachineConfigOperator),
 		"05-openshift-cluster-api-namespace.yaml":          []byte(bootkube.OpenshiftClusterAPINamespace),
+		"09-openshift-service-signer-namespace.yaml":       []byte(bootkube.OpenshiftServiceCertSignerNamespace),
```

Comment:
I used '09-' to match the operator's runlevel here, but I'm not sure if that is necessary.